src/platform/web/ui/general/StaticView.ts

```diff
@@ -15,15 +15,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import {tag} from "../general/html";
+import {tag} from "./html";
 
 export class StaticView {
-    constructor(value, render = undefined) {
+    private _root?: Element;
+    render: any;
```

Comment:
```suggestion
    render: (typeof tag, T) => Element;
```

Reply:
> 

This throws in the error `This expression is not callable. Type 'never' has no call signatures.ts(2349)` for line 28. Using typeof also throws in error.

Reply:
> 

I'll look into it. I had to use `any` in both line 22 and 23 to avoid any error. :sweat_smile: So I can't think of assigning any other type to render.